code/obj/flock/structure/compute.dm

```diff
@@ -0,0 +1,29 @@
+/obj/flock_structure/compute
+	name = "weird lookin' thinking thing"
+	desc = "It almost looks like a terminal of some kind."
+	flock_id = "Compute node"
+	health = 60
+	icon_state = "compute"
+	compute = 60
+	var/static/display_count = 9
```

Comment:
It's better to do `#define MAXIMUM_LIFE_SPRITE_NUMBER 9` instead of an actual variable